config/routes.rb

```diff
@@ -44,6 +44,8 @@
   end
   get 'cockpit/redirect', to: 'cockpit#redirect'
 
+  match 'job_wizard', to: 'remote_execution_react#index', :via => [:get]
```

Comment:
if this is an experimental feature, don't we want to have `experimental` 'prefix'  in the route?
i.e `/experimental/job_wizard`

Reply:
Added the prefix